RadixWallet/Features/ImportMnemonic/ImportMnemonic.swift

```diff
@@ -299,12 +316,15 @@ public struct ImportMnemonic: Sendable, FeatureReducer {
 		}
 
 		public var body: some Reducer<State, Action> {
-			Scope(state: /State.markMnemonicAsBackedUp, action: /Action.markMnemonicAsBackedUp) {
+			Scope(state: /State.backupConfimartion, action: /Action.backupConfimartion) {
 				EmptyReducer()
```

Comment:
The scoping with EmptyReducer can be removed completely. See recent Slack thread.